packages/spread/operation/add_stop_to_nutrient.py

```diff
@@ -0,0 +1,54 @@
+from neo4j import GraphDatabase
+from pydantic import BaseModel
+
+from spread.operation.framework import Operation
+
+class EngagementData(BaseModel):
+    region_id: str
+
+class AddStopToNutrient(Operation):
+    # TODO: Add class description.
```

Comment:
Let's get rid of the TODO's for now - might be worth creating a Jira ticket for class/method descriptions if you feel strongly about it

Reply:
Removed for now, can clean up TODOs before they reach a PR.